wordpress-mobile/WordPress-iOS
# [Gutenberg] Upload React Native bundle and source map files from Gutenberg framework

**Related PRs:**
* https://github.com/wordpress-mobile/gutenberg-mobile/pull/6760

**To test:**
1. Run command: `bundle exec pod install`.
2. Observe that the source map file (`App.composed.js.map` or `main.jsbundle.map`) is not present in the folder `Pods/Gutenberg/Frameworks/Gutenberg.xcframework/ios-arm64/Gutenberg.framework`.
4. Download the installable build and inspect the IPA file.
5. Observe that the source map file (`App.composed.js.map` or `main.jsbundle.map`) is not present, only the file `main.jsbundle`.
6. Observe in the CI job `Jetpack Prototype Build` or `WordPress Prototype Build`, that the step `sentry_upload_sourcemap ` uploads the bundle and source map files to Sentry.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. 
What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


Testing checklist:
